apps/website/src/data/found-animal.ts

```diff
@@ -242,6 +242,196 @@ const data: FoundAnimalFlow = {
+                            "Do not attempt to feed or give water to the squirrel, as this may lead to it imprinting on humans and becoming dependent on them.",
+                            "Return the squirrel to its nesting tree (usually in the immediate area where the squirrel was found). If you're unsure of the right tree, or if the nest was destroyed, choose a tree closest to where the squirrel was found.",
+                            "Observe the baby for the next 6-8 hours, reheating the rice every two hours or so to keep it warm.",
+                            "Has the mother returned?",
+                          ],
+                          options: [
+                            {
+                              name: "Yes",
+                              flow: {
+                                prompt: macros.general.leave("the squirrel"),
+                              },
+                            },
+                            {
+                              name: "No",
+                              flow: {
+                                prompt: macros.general.rehab(
+                                  "the squirrel",
+                                  "If you are sure the parents are not nearby, and do not return within a few hours",
+                                ),
+                              },
+                            },
+                          ],
+                        },
+                      },
+                      {
+                        name: "No",
+                        flow: {
+                          prompt: macros.general.leave("the squirrel"),
+                        },
+                      },
+                    ],
+                  },
+                },
+              ],
+            },
+          },
+        ],
+      },
+    },
+    {
+      name: "Raccoon",
+      flow: {
+        prompt: ["Does the raccoon appear to be sick or injured?"],
+        options: [
+          {
+            name: "Yes",
+            flow: {
+              prompt: [
+                "Is the raccoon an adult or a baby?",
+                "Baby raccoons are normally between 4 and 14 inches in length, whereas a fully grown adult can be up to 3 feet long. Babies are often extremely vocal, growling, hissing, and snorting if disturbed.",
+              ],
+              options: [
+                {
+                  name: "Adult",
+                  flow: {
+                    prompt: [
+                      "If you have found an injured adult raccoon, although it may be hard to accept, adult raccoons can rarely be caught and successfully treated.",
+                      "If you see an adult raccoon with an injury, leave the animal alone. Even though the injury may take a long time to heal, this is far preferable to the trauma of chase and capture.",
+                    ],
```

Comment:
Instead of giving this advice, it would be preferred to use the rehab macro here.